cryptoshred-spring-boot-autoconfigure/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>eu.prismacapacity</groupId>
+    <artifactId>cryptoshred</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>cryptoshred-spring-boot-autoconfiguration</artifactId>
+  <properties>
+    <spring-boot.version>2.2.4.RELEASE</spring-boot.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${spring-boot.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-context</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-autoconfigure</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>eu.prismacapacity</groupId>
+      <artifactId>cryptoshred-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>eu.prismacapacity</groupId>
+      <artifactId>cryptoshred-spring</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>eu.prismacapacity</groupId>
```

Comment:
Is this really necessary? 🤔

Reply:
yes, i think so. please elaborate. Note that transitive does not work for optional deps.

Reply:
Mh yes, but on the client site you have to define a dependency to `cryptoshred-spring-cloud-aws` anyways which includes `cryptoshred-cloud-aws` so we should be fine without it, shouldnt we?

If not I guess we need a dependency to `cryptoshred-micrometer` as well?! 🤔

Reply:
good point.